src/backend-storage/mde-mode-api.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Sets the status of whether to enable markdown editing
+ * @param {Boolean} statusMDE
+ */
```

Comment:
If you are adding @name parameters to the functions, they need to be added here.

Reply:
I think we are only adding name parameters to anonymous window onload functions